crates/cli/src/bin/wasm-bindgen.rs

```diff
@@ -110,6 +110,7 @@ fn rmain(args: &Args) -> Result<(), Error> {
             "nodejs" => b.nodejs(true)?,
             "deno" => b.deno(true)?,
             "experimental-nodejs-module" => b.nodejs_module(true)?,
+            "isomorphic" => b.isomorphic(true)?,
```

Comment:
Since this is a proof-of-concept I think `experimental-isomorphic` for the name makes more sense.

Reply:
- [ ] change cli option